frontend/src/components/MapView/LeftPanel/AnalysisPanel/AnalysisTable/ExposureAnalysisTable/index.tsx

```diff
@@ -197,7 +197,7 @@ const ExposureAnalysisTable = memo(
           rowsPerPage={rowsPerPage}
           page={page}
           onPageChange={handleChangePage}
-          onChangeRowsPerPage={handleChangeRowsPerPage}
+          onRowsPerPageChange={handleChangeRowsPerPage}
```

Comment:
why the name change?

Reply:
That was a warning from the upgrade of the Materia UI the `onChangeRowsPerPage` was deprecated in favor of `onRowsPerPageChange`  so I changed it to stop the console errors from Material